src/othermbc.md

```diff
@@ -40,70 +41,75 @@ nonsensical.
 
 The mapper does not support an outer bank for battery SRAM.
 
-To start a game, do the following in code run from RAM: Write $A5 to
-$1000, write game starting bank number to $2000, write any value to
-$7000, write $98 to $1000, write $01 to $2000 (so that 32K games
-work), jump to $0100.
+To start a game, do the following with code run from RAM:
```

Comment:
```suggestion
To start a game, execute the following steps with code running from RAM:
```

Reply:
```suggestion
To start a game, do the following with code running from RAM:
```
or
```suggestion
To start a game, perform the following steps with code running from RAM:
```

Since all the steps are already formulated in imperative tense, I think "do the following" makes more sense than using the verb "execute", which implies code.

Reply:
Second one works for me